steps/unzip/0.0.1/step.yml

```diff
@@ -0,0 +1,37 @@
+title: Unzip
+summary: |
+  Unzip an archive into a directory
+description: |
+  This step takes a zip file as input and unzips its content to a directory.
+website: https://github.com/digitalrmdy/bitrise-step-unzip
+source_code_url: https://github.com/digitalrmdy/bitrise-step-unzip
+support_url: https://github.com/digitalrmdy/bitrise-step-unzip/issues
+published_at: 2019-10-29T13:21:40.837467+01:00
+source:
+  git: https://github.com/digitalrmdy/bitrise-step-unzip.git
+  commit: ea2789a79afc0db53c081dd862406dda919cad28
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
```

Comment:
Variables here: https://github.com/digitalrmdy/bitrise-step-unzip/blob/master/step.sh#L4
should be enclosed in `"`.
Currently script won't work as expected if variables contain spaces.

Reply:
Fixed in #2223